root.sh

```diff
@@ -107,6 +107,7 @@ else
         -Dsoversion=ON                                            \
         -Dshadowpw=OFF                                            \
         -Dvdt=ON                                                  \
+        -Dbuiltin_vdt=ON                                          \
         -DCMAKE_PREFIX_PATH="$FREETYPE_ROOT;$SYS_OPENSSL_ROOT;$GSL_ROOT;$ALIEN_RUNTIME_ROOT;$PYTHON_ROOT;$PYTHON_MODULES_ROOT"
   FEATURES="builtin_pcre mathmore xml ssl opengl minuit2 http
             pythia6 roofit soversion vdt ${CXX11:+cxx11} ${CXX14:+cxx14} ${XROOTD_ROOT:+xrootd}
```

Comment:
Hi, in principle you should also change this line in order to let the recipe check for `builtin_vdt` instead of `vdt`. Can you try to do that, and check whether it works on ROOT 5 and ROOT 6?